nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/util/JdbcCommon.java

```diff
@@ -300,7 +307,7 @@ public static Schema createSchema(final ResultSet rs, String recordName, boolean
                     // Check the precision of the BIGINT. Some databases allow arbitrary precision (> 19), but Avro won't handle that.
                     // If the precision > 19 (or is negative), use a string for the type, otherwise use a long. The object(s) will be converted
                     // to strings as necessary
-                    int precision = meta.getPrecision(i);
+                    int precision = dbAdapter.getSupportsMetaDataColumnGetPrecision()?meta.getPrecision(i):0;
```

Comment:
Probably `-1` would be a safer choice. If we can't retrieve precision, and especially since HIVE supports up to Decimal 38... we don't want to accidentally lose data or start throwing exceptions for overflows.